docs/docs/guides/integrate/identity-providers/github.md

```diff
@@ -0,0 +1,101 @@
+
+## GitHub Configuration
+
+### Register a new application
+
+For **GitHub** browse to the [Register a new OAuth application](https://github.com/settings/applications/new). You can find this link withing [Settings](https://github.com/settings/profile) - [Developer Settings](https://github.com/settings/apps) - - [OAuth Apps](https://github.com/settings/developers).
+
+For **GitHub Enterprise** go to your GitHub Enterprise home page and then to Settings - Developer Settings - OAuth Apps - Register a new application/New OAuth App
+
+Fill in the application name and homepage URL.
+
+You have to add the authorization callback URL, where GitHub should redirect, after the user has authenticated himself.
+In this example our test instance has the domain `https://acme-gzoe4x.zitadel.cloud`.
+This results in the following authorization callback URL:
+ `https://acme-gzoe4x.zitadel.cloud/ui/login/login/externalidp/callback`
+
+:::info
+To adapt this for you setup just replace the domain
+:::
+
+![Register an OAuth application](/img/guides/github_oauth_app_registration.png)
+
+### Client ID and Secret
+
+After clicking "Register application", you will see the detail page of the application you have just created.
+Copy the client ID you can copy directly from the detail page. 
+Generate a new secret by clicking "Generate new client secret".
+Make sure to save the secret, as you will not be able to show it again.
+
+![Client ID and Secret](/img/guides/github_oauth_client_id_secret.png)
+
+## ZITADEL Configuration
+
+### Create new GitHub Provider
+
+Go to the settings of your ZITADEL instance or the organization where you like to add a new GitHub provider.
+Choose the GitHub provider template. This template has everything you need preconfigured. You only have to add the client ID and secret, you have created in the step before.
+
+You can configure the following settings if you like, a useful default will be filled if you don't change anything:
+
+**Scopes**: The scopes define which scopes will be sent to the provider, `openid`, `profile`, and `email` are prefilled. This informations will be taken to create/update the user within ZITADEL.
+
+<GeneralConfigDescription name="GeneralConfigDescription" />
+
+![GitHub Provider](/img/guides/zitadel_github_create_provider.png)
+
+### Activate IdP
+
+Once you created the IdP you need to activate it, to make it usable for your users.
+
+![Activate the GitHub](/img/guides/zitadel_activate_github.png)
+
+## Test the setup
+
+To test the setup use incognito mode and browse to your login page.
+If you succeeded you should see a new button which should redirect you to your GitHub.
+
+Per default the login of your instance will be shown, read the following section on how to trigger it for a specific organization: [Organization Scope](./general#trigger-configuration-on-the-login-for-a-specific-organization)
```

Comment:
Can we centralize this?

Reply:
comment is obsolete if we implement https://github.com/zitadel/zitadel/pull/5565#pullrequestreview-1374481217